examples/fmm-error.py

```diff
@@ -89,7 +95,7 @@ def main():
     pt.gca().yaxis.set_major_formatter(NullFormatter())
 
     cb = pt.colorbar(shrink=0.9)
-    cb.set_label(r"$\log_{10}(\mathdefault{Error})$")
+    cb.set_label(r"$\log_{10}(\mathrm{Error})$")
```

Comment:
A little justification: I get an error here
```
RuntimeError: latex was not able to process the following string:
b'$\\\\log_{10}(\\\\mathdefault{Error})$'
```
Not sure if my setup is weird or what, since `mathdefault` should be fine from what the docs say.